ssg/constants.py

```diff
@@ -383,6 +383,7 @@
     "shadow-utils": "cpe:/a:shadow-utils",
     "systemd": "cpe:/a:systemd",
     "yum": "cpe:/a:yum",
+    "sssd": "cpe:/a:sssd",
```

Comment:
nitpick: Please keep the CPEs ordered alphabetically after `machine` and `container` platform CPEs.

Reply:
Fixed in 8fb9b8c2cfeb7659952fab86a69739ff9f05edd6.